03-router.md

```diff
@@ -137,13 +137,13 @@ You guessed it! Still inside `menu_get_item()`.
 
 This part is more confusing than you may expect. 
 
-It's confusing first of all because it happens in a function called [`_menu_translate()`](https://api.drupal.org/api/drupal/includes%21menu.inc/function/_menu_translate/7) which in the case, means "translate" as in "translate placeholders in the URL to loaded entities (or other things), when needed". (It also translates the menu title to the appropriate language later in this same function, confusing things further).
+It's confusing first of all because it happens in a function called [`_menu_translate()`](https://api.drupal.org/api/drupal/includes%21menu.inc/function/_menu_translate/7) which means "translate" as in "translate placeholders in the URL to loaded entities (or other things), when needed". (It also translates the menu title to the appropriate language later in this same function, confusing things further).
 
 To start, it needs to translate placeholders (i.e., "%" signs) in paths to the loaded entity if a load function exists for it. For example, if our path in `menu_router` is `node/%node` then this chunk of code will say "ok, I see `%node` as opposed to just `%`, therefore, I know I need to run `node_load()` on whatever is in that part of the URL, which in this case is `1234`. 
 
-To do this, it runs [`_menu_load_objects()`](https://api.drupal.org/api/drupal/includes%21menu.inc/function/_menu_load_objects/7) which calls the appropriate load function if there is one and, assuming we don't get an access error or a missing entity or anything else which would throw an error, put the result (i.e., the fully loaded entity) back into the menu router item for later use. 
+To do this, it runs [`_menu_load_objects()`](https://api.drupal.org/api/drupal/includes%21menu.inc/function/_menu_load_objects/7) which calls the appropriate load function if there is one and, assuming we don't get an access error, or a missing entity, or anything else which would throw an error, put the result (i.e., the fully loaded entity) back into the menu router item for later use. 
```

Comment:
If you're going to do a serial list, you need to omit the or immediately after access error. What Mike has is grammatically correct, but I think it reads better as a serial list.